### PR TITLE
LPS-121892 Migrate v2 usages in my-subscriptions/my-subscriptions-web

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @liferay-lima,

Attached is an update for http://issues.liferay.com/browse/LPS-121892.

**Test Plan:**

1. Login to Liferay
2. Click on the **Control Menu** -> **Content & Data** -> **Message Boards**
3. Click on the `+` to add a new **Thread**
4. Publish a new message board and make sure **Subscribe Me** is checked under **More Settings**
5. Click on the **User Icon** -> **My Subscriptions**

my-subscriptions

Please let me know if you have any questions. Thanks!

### CODE REVIEW

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by boton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/my-subscriptions/my-subscriptions-web/src/main/resources/META-INF/resources/js/MySubscriptionsManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			if (item.data?.action === 'unsubscribe') {
+				const form = document.getElementById(`${portletNamespace}fm`);
+
+				if (form) {
+					form.setAttribute('method', 'post');
+
+					var subscriptionIds = form.querySelector(
```
Comment (boton) [inline modules/apps/my-subscriptions/my-subscriptions-web/src/main/resources/META-INF/resources/js/MySubscriptionsManagementToolbarPropsTransformer.js:25]: Maybe a `const` fit better

**Comment by boton** [comment]
We lose the `Select Items` label

image

But I think that it will be fixed by @ambrinchaudhary in #1507

**Comment by boton** [comment]
LGTM, except var detail

**Review by carloslancha — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
> But I think that it will be fixed by @ambrinchaudhary in #1507

You are right!

File: modules/apps/my-subscriptions/my-subscriptions-web/src/main/resources/META-INF/resources/js/MySubscriptionsManagementToolbarPropsTransformer.js (hunk 1 above)
Comment (jonmak08) [reply modules/apps/my-subscriptions/my-subscriptions-web/src/main/resources/META-INF/resources/js/MySubscriptionsManagementToolbarPropsTransformer.js]: Hey @boton,

That change has been made and pushed up with https://github.com/liferay-lima/liferay-portal/pull/1520/commits/5f82ad0f0fe668dd6e40e87157a696f5ddacae1e.

Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**